### ISSUE TITLE
Video Quality Option

### ISSUE THREAD

**Issue by brunocalado** [bekriebel/fvtt-module-avclient-livekit#54]
Is it possible to add a video quality option to reduce bandwidth consumption?

![image](https://github.com/bekriebel/fvtt-module-avclient-livekit/assets/4999783/59d9f77b-b3bd-43df-b887-9fca489707bc)

tks

**Comment by jacksands** [comment]
pls! this would be really awesome especially for gaming transmissions were camera thumb is quite small and need not great quality, reducing any lag and charges when using the paid service of Livekit server!!

and more users connected, more pronounced the difference would be!!


and btw   tks for the amazing module!  really helpful!

**Comment by bekriebel** [comment]
Yeah, I've been swamped lately and just recently got the client updated to the latest versions of LiveKit. My hope for this week is to go through issues and PRs and start getting things like that added in :-)

**Comment by Fridan99** [comment]
Any news about this?
